beacon-chain/core/light-client/store.go

```diff
@@ -92,7 +206,17 @@ func (s *Store) LightClientUpdates(ctx context.Context, startPeriod, endPeriod u
 		return nil, err
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "failed to get updates from the database")
```
Because you have a similar error wrap below